extensions/Worker.Extensions.Abstractions/release_notes.md

```diff
@@ -0,0 +1,5 @@
+## Release notes
+<!-- Please add your release notes in the following format:
+- My change description (#PR/#issue)
+-->
```

Comment:
Can you update the format here to match what's in the repo root? Would help ensure the release notes are consistent.